fr.txt

```diff
@@ -0,0 +1,405 @@
+ServerName = "Nom du serveur"
+ServerNameDescription = "Ajouter un serveur vous donne un jeton d'accès qui vous permet de lancer votre propre serveur dédié."
+NoServersDescription = "Aucun serveur associé à ce jeu sur votre compte."
+AddNewServer = "Ajouter un nouveau serveur"
+AddingServer = "Ajout du serveur…"
+GameServerWasGenerated = "Le serveur a été ajouté avec succès."
+RemoveServer = "Supprimer"
+ConfigureServer = "Configurer"
+ConfigureServerTitle = "Configurer le serveur"
+ConfigureServerDescription = "Les champs suivants vous permettent de configurer le serveur. Suivant votre système d'exploitation, vous devrez aussi exécuter quelques scripts. Merci de suivre les instructions correspondant à votre machine: <a href='https://forums.kleientertainment.com/forums/topic/64212-dedicated-server-quick-setup-guide-windows/' target='_blank'>Windows</a> &mdash; <a href='https://forums.kleientertainment.com/forums/topic/64441-dedicated-server-quick-setup-guide-linux/' target='_blank'>Linux</a> &mdash; <a href='https://forums.kleientertainment.com/forums/topic/64687-dedicated-server-quick-setup-guide-mac-os-x/' target='_blank'>macOS</a>."
+DownloadSettings = "Télécharger les paramètres"
+FriendlyName = "Alias"
+
+; Account Betas
+; https://accounts.klei.com/account/betas
+
+UpcomingBetasTitle = "bêtas actives et à venir"
+UpcomingBetasDescription = "Vous pouvez vous inscrire aux bêtas qui vous intéressent. Vous recevrez un courriel lorsque vous serez accepté."
+BetaStatus = "Statut:"
+BetaStartDate = "Début:"
+BetaRegister = "Inscription"
+BetaRegistered = "Inscrit"
+BetaNotAccepted = "Inscrit - non accepté"
+BetaAccepted = "Accepté"
+BetaRegistrationEnded = "Inscriptions terminées"
+
+; Community Translations
+; https://accounts.klei.com/community-translations
+
+CommunityTranslations = "Traductions"
+CommunityTranslationDisclaimer = "<a href='/community-translations'>Traduction non officielle.</a>"
+CommunityTranslationsDescription = "Si vous voulez ajouter une nouvelle langue ou faire des corrections <a href='https://github.com/kleientertainment/KleiAccountsLocale' target='_blank' rel='noopener noreferrer'>suivez ce lien</a> et lisez les instructions indiquant comment contribuer à la traduction de ce site web."
+
+; Rewards
+; https://accounts.klei.com/account/rewards
+
+RewardTitle = "Drops & Récompenses (bêta)"
+RewardDescription = "Restez à jour et gagnez des récompenses dans le jeu. Nous ajouterons des récompenses de temps en temps, donc vérifiez souvent s'il y a du nouveau!"
+RewardLabel = "Récompenses"
+RewardStartButton = "Relier un compte"
+RewardClaimButton = "Demander une récompense"
+RewardWasClaimed = "Demande réussie!"
+RewardWasRedeemed = "Échange réussi!"
+RewardAlternative = "Alternative"
+RewardPointsAbbreviation = "pts"
+RewardPointsMinutes = "min"
+RewardBalance = "Valeur actuelle"
+RewardNameTwitch = "Regarder “Don’t Starve Together” sur Twitch"
+RewardNameDouyu = "Regarder “Don’t Starve Together” sur DouYu"

```

Comment:
Have to decide if we use infinitive or imperative:
Here we have `RegardeR` but then we use `SuiveZ-nous`.

Reply:
To check in context